frontend/src/pages/modelServing/screens/projects/ServingRuntimeTokenDisplay.tsx

```diff
@@ -21,7 +21,7 @@ const ServingRuntimeTokenDisplay: React.FC<ServingRuntimeTokenDisplayProps> = ({
     return (
       <HelperText>
         <HelperTextItem variant="warning" hasIcon>
-          Failed to get token. {error && error.message}
+          Failed to get token. {error?.message}
```

Comment:
`error?.message` I don't think you need the `?` here, maybe remove it?

Reply:
I think is needed, error could be undefined in this scenario.